I'm sorry, I forgot to copy the compiler warning. I looked at a recent AMD64 build on Windows buildbots:

```
d:\buildarea\3.x.bolen-windows10\build\python\ast.c(4316): warning C4244: '+=': conversion from '__int64' to 'int', possible loss of data [D:\buildarea\3.x.bolen-windows10\build\PCbuild\pythoncore.vcxproj]
```

ast.c:4316:
```
    int cols = parent->n_col_offset;
(...)
            cols += substr - start;  // <--- HERE
```

--- a/Python/ast.c
+++ b/Python/ast.c
@@ -4313,7 +4313,7 @@ fstring_fix_node_location(const node *parent, node *n, char *expr_str)
                     break;
                 start--;
             }
-            cols += substr - start;
+            cols += (int)(substr - start);
             /* Fix lineno in mulitline strings. */
             while ((substr = strchr(substr + 1, '\n')))
                 lines--;

